src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp

```diff
@@ -182,32 +204,50 @@ JitConstants EltwiseKernel_blocked_opt::MakeLoadJitConstants(const eltwise_param
                                                         "(input" + toCodeString(input.index) + "[0])";
                         jit.AddConstant(MakeJitConstant(vload_name, vload_value));
                         jit.AddConstant(MakeJitConstant(name, "tmp_a" + toCodeString(op_num) + "_" + toCodeString(input_idx)));
```

Comment:
const std::string temp_vec_var = "\\\n\tMAKE_VECTOR_TYPE(ACCUMULATOR_TYPE, " + toCodeString(vec_size) + ")" +
                                                     " tmp_a" + toCodeString(op_num) + "_" + toCodeString(input_idx) + " = ";

Reply:
What about introducing `MAKE_VECTOR_TYPE(ACCUMULATOR_TYPE, " + toCodeString(vec_size) + ")` as a separate variable? It is used multiple times so that will make the code more readable..

Reply:
Applied